class.jetpack-gutenberg.php

```diff
@@ -45,6 +62,22 @@ function jetpack_register_plugin( $slug ) {
  *
  * @return void
  */
+function jetpack_set_extension_unavailable( $slug, $reason ) {
+	Jetpack_Gutenberg::set_extension_unavailable( $slug, $reason );
```

Comment:
this is tangential to this PR, but why do we have so many wrapper functions like this? why don't we call `Jetpack_Gutenberg::set_extension_unavailable()` directly?

Reply:
Beats me. I'm doing a bit of cargo cult following here since "it's been like this before":tm:, but I don't really like them either.

@enejb Think it's fine to drop these wrappers?

Reply:
The main idea behind the wrapper functions is that we have simple functions to call and the underling classes would change but the function calls would stay as they are. 

Kind of like what core does with `add_action`, `do_action` function calls for example. 

In the end I think we should not be removing functions we shipped since someone could be using them for some reason. 

> Think it's fine to drop these wrappers?

What is the argument for dropping them?

Reply:
> What is the argument for dropping them?

I'm just asking what the argument was for creating them. I'd be more inclined to use the inner wrapped function directly until the need arises to change something and we know we'll need to. Those times may be difficult because we'll have to find and replace those function calls, but they may not come either.

I'd be suspicious about the claim that we can change the underlying classes though without also changing this wrapper, because if there are any behavioral changes involved we'd have to account for that in some case.

---

Should we consider dropping the wrapper for now? Would anyone be opposed to doing that?

Reply:
I'm more than happy to drop them.

@enejb From experience, these synonymous (or near-synonymous) wrappers have been kind of a pain to update when the API changed (e.g. function signature changes requiring the introduction of new/different functions). I think it's fine to make `Jetpack_Gutenberg::something()` the interface (as opposed to `jetpack_something()` -- after all, these _are_ JP/GB specific things, so we don't need to hide that from consumers IMO.

Reply:
I am fine with dropping the wrapper as long as we deprecated it properly. Or we are confident that it will not break the sites out there.

Reply:
I've dropped the wrappers that I'd newly written as part of this PR altogether, and deprecated the others.